SU-SWS/stanford_basic
# D8CORE-1352: Prevent javascript active path highlighting on secondary nav

# READY FOR REVIEW

# Summary
- Fixes active path bug when you're on a top level page and the child was highlighted.

# Review By (Date)
- End of the week

# Urgency
- Highish

# Steps to Test

1. Check out this branch
2. Create a stanford_page under a top level item. eg: Research -> More Research
3. Navigate to the top level page by clicking on the link in the top (primary) navigation
4. Ensure the secondary page is not highlighted when the page loads
5. Click on secondary nav page and ensure it is highlighted when the page loads.

# Affected Projects or Products
- Stanford Basic

# Associated Issues and/or People
- D8CORE-1352

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
